[2022-03-18 19:48:49,394] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 19:48:49,436] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2022-03-17T00:00:00+00:00 [queued]>
[2022-03-18 19:48:49,436] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 19:48:49,436] {taskinstance.py:1068} INFO - Starting attempt 4 of 4
[2022-03-18 19:48:49,437] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 19:48:49,471] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2022-03-17T00:00:00+00:00
[2022-03-18 19:48:49,478] {standard_task_runner.py:52} INFO - Started process 26329 to run task
[2022-03-18 19:48:49,484] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2022-03-17T00:00:00+00:00', '--job-id', '20', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp1v3c5x7s', '--error-file', '/tmp/tmpy4e6z054']
[2022-03-18 19:48:49,487] {standard_task_runner.py:77} INFO - Job 20: Subtask task_1
[2022-03-18 19:48:49,568] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2022-03-17T00:00:00+00:00 [running]> on host pc
[2022-03-18 19:48:49,691] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2022-03-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-17T00:00:00+00:00
[2022-03-18 19:48:49,695] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-03-18 19:48:49,696] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-03-18 19:48:49,705] {subprocess.py:75} INFO - Output:
[2022-03-18 19:48:52,708] {subprocess.py:83} INFO - Command exited with return code 0
[2022-03-18 19:48:52,825] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20220317T000000, start_date=20220318T141849, end_date=20220318T141852
[2022-03-18 19:48:52,903] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-18 19:48:52,908] {local_task_job.py:151} INFO - Task exited with return code 0
